stable/appmesh-manager/templates/service.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "appmesh-manager.fullname" . }}-metrics-service
+  namespace: {{ .Release.Namespace }}
+  labels:
+    control-plane: controller-manager
+{{ include "appmesh-manager.labels" . | indent 4 }}
+spec:
+  ports:
+   - name: https
+     port: 8443
+     targetPort: https
+  selector:
+    control-plane: controller-manager
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "appmesh-manager.fullname" . }}-webhook-service
+  namespace: {{ .Release.Namespace }}
+  labels:
+{{ include "appmesh-manager.labels" . | indent 4 }}
+spec:
+  ports:
+  - port: 443
+    targetPort: 9443
+  selector:
+    control-plane: controller-manager
```

Comment:
We shouldn't use a generic label as this can conflict with other kubebuilder controllers, I propose we set the selector to `{{ template "appmesh-manager.fullname" . }}`.

Reply:
got it. thanks